src/common/options/global.yaml.in

```diff
@@ -2070,6 +2070,16 @@ options:
   services:
   - mon
   with_legacy: true
+# 主动释放内存
```

Comment:
@jdurgin: Is Simplified Chinese allowed in comments in the Ceph project?

Reply:
comments should be in english